### DISCUSSION TITLE
The ability to zoom in when comparing images.

### DISCUSSION THREAD

**Discussion by OdraSTM** [qarmin/czkawka#982] [Ideas]
Thank you for creating this software.

Before this I would use mostly use Visipics to do most of the work when eliminating dupes, and I was fond of its ability to zoom in to gauge artifacts on compressed images. Now I could simply gauge quality by size and image formant, but internet shenanigans can often result in things such as a mediocre Jpeg converted into a large PNG...

I like the side by side comparison and would like to put forth the idea of being able to zoom in during that comparison, so both images would zoom in to the same area (or the approximate location based on positional percentage rather than pixels in case of different dimensions) while also offering to render the image with different scaling filtering methods (Nearest/Lancos)

Thanks for the consideration

**Comment by Alister-Zeus** [comment]
Size-by-side is a great option especially when it can be in a separate window, as I have 3 monitors

Most image professionals work on at least two, with three being great

- Working area
- tools
- whole image

The later one is to avoid the old 3 arms blunder
